README.md

```diff
@@ -23,16 +23,22 @@ Please download the latest developer bundle from [here](https://developer.aevi.c
 
 ## Integrate
 
-AppFlow and its dependencies are as of `v2.1.0` published to the `jcenter()` repository which is used by default by Android Gradle project / Android Studio.
+AppFlow and its dependencies are as of `v2.2.4` published to the Github packages repository. This is in response to `bintray` and `jcenter` shutting down in May 2021.
+Version `2.2.4` is functionally equivalent to the last published `jcenter()` version (`2.2.3`).
 
-If you are using an earlier version or having issues getting this to work, you can also add our bintray repository directly as per below. 
+If you require access to an earlier version once `jcenter` has shut down, please contact AEVI for assistance.
 
 In your root project `build.gradle` file, add
 
 ```
 repositories {
     maven {
-        url "http://dl.bintray.com/aevi/aevi-uk"
+        name = "AEVI-repos"
+        url = uri("https://maven.pkg.github.com/aevi-appflow/*")
+        credentials {
+            username = <your Github username>
```

Comment:
Right, so should we create a GH user and provide details here? Bit indecisive...

Reply:
Why would we need to? Surely that's down to the developer using it?

Reply:
Yeah but there will be complaints of course as that is a major pain, especially for CI. Not everyone will have a Github "robot" and would then have to use private Github details in CI etc. It's not very user friendly